docs/config/xmlConfigurationFiles.rst

```diff
@@ -128,17 +128,31 @@ The configuration value itself is defined as a cvalue element. Each element can
 * *type* - There exist several types for configuration values. The default value is string. Note, that the selection of
   the type will influence, how variables are interpreted and evaluated / converted.
 
-  - *string* will
+  - *string* accepts any value.
 
   - *int* will accept integer values only. E.g. 1, 2, 3 or 4.
 
-  - *float* will accept float values in differnt formats. E.g. 1.2f 1.2
+  - *float* will accept valid Java / Groovy float values. E.g. 1.2f 1.2
 
-  - *double* will accept
+  - *double* will accept valid Java / Groovy double values. E.g. 1.2 or 1.2E-3.
 
+  - *boolean* will evaluate true (preferred), y, j, t and 1 to true and false (preferred), n, f and 0 to false.
 
+  - *path* indicates, that the variable is a path or a part of a path.
 
-  - *boolean*
+Nice to know - Order of evaluation in Roddy
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+Configuration values can be cast to their *type* by calling any of the methods *toBoolean()*,
```

Comment:
Do the users of the XML know anything about the toFloat() etc. methods? Do they need to know?
Really focus on the declared type "float", "path", "file", etc. and explain the variable evaluation mechanism. Leave out Roddy interna.